### PR TITLE
Added french language to i18n.yml

### PR DESCRIPTION
**Pull request by slashformotion**
The date formating was adapted to french habits.

### CODE REVIEW

File: _data/i18n.yml [hunk 1]
```diff
@@ -52,3 +52,19 @@ ko:
...
+    abbr_day_names: ["Dim", "Lun", "Mar", "Mer", "Jeu", "Ven", "Sam"]
+    abbr_month_names: ["Jan", "Fev", "Mar", "Avr", "Mai", "Jui",
+      "Jui", "Aou", "Sep", "Oct", "Nov", "Dec"]
+    day_names: ["Dimanche", "Lundi", "Mardi", "Mercredi", "Jeudi", "Vendredi", "Samedi"]
+    month_names: ["Janvier", "Février", "Mars", "Avril", "Mai", "Juin",
+      "Juillet", "Aout", "Septembre", "Octobre", "Novembre", "Décembre"]
+  morelink: "Lire plus"
+  page-older: "Plus ancien"
+  page-newer: "Récent"
+  archive: "Archives"
```
Comment (yous) [inline _data/i18n.yml:67]: This seems to be a translation of "Archive." Can you update to match with "Blog Archive?" If this is correct, please let me know.

**Review by yous — COMMENTED** [review]
Thanks for the contribution! Just one comment.

File: _data/i18n.yml (hunk 1 above)
Comment (slashformotion) [reply _data/i18n.yml:67]: I think it's better to stick with this one. You are right, "Archives du blog" would be a better translation but it's quite long. And in french, "Archives" is very understandable.

Comment (yous) [reply _data/i18n.yml:67]: Okay, I get it. Thanks for the explanation.

**Pull request merged**